pkg/reconciler/messaging/channel/channel.go

```diff
@@ -104,21 +103,24 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, channel *v1alpha1.Channe
 	return pkgreconciler.NewEvent(corev1.EventTypeNormal, reconciledSuccessReason, `Channel reconciled: "%s/%s"`, channel.Namespace, channel.Name)
 }
 
-func (r *Reconciler) syncSubscribers(ctx context.Context, channel *v1alpha1.Channel) error {
-	if channel.Status.SubscribableStatus == nil {
-		channel.Status.SubscribableStatus = &eventingduck.SubscribableStatus{
-			Subscribers: make([]eventingduckv1beta1.SubscriberStatus, 0),
-		}
-	} else if channel.Status.SubscribableStatus.Subscribers == nil {
+func (r *Reconciler) syncSubscribers(ctx context.Context, channel *v1beta1.Channel) error {
+	//if equality.Semantic.DeepEqual(channel.Status.SubscribableStatus, eventingduckv1beta1.SubscribableStatus{}) {
+	//	channel.Status.SubscribableStatus = eventingduckv1beta1.SubscribableStatus{
+	//		Subscribers: make([]eventingduckv1beta1.SubscriberStatus, 0),
+	//	}
+	//} else if channel.Status.SubscribableStatus.Subscribers == nil {
+	//	channel.Status.SubscribableStatus.Subscribers = make([]eventingduckv1beta1.SubscriberStatus, 0)
+	//}
+	if channel.Status.SubscribableStatus.Subscribers == nil {
```

Comment:
Not sure whether we need this check.
Since we use https://github.com/google/knative-gcp/blob/94e9331b180d315d058ea867b8e368096f807e80/pkg/client/injection/reconciler/messaging/v1beta1/channel/reconciler.go#L184 to compare the status,
whether it's nil or empty doesn't matter

Reply:
Can you get a nil pointer somewhere below if you don't check this? 
I think i'd rather keep it...